### PR TITLE
[tests-only] Ensure compatibility with PHPUnit v9.5

### PR DESCRIPTION
**Pull request by JammingBen**
Fixes https://github.com/owncloud/files_external_s3/issues/746

See comments below - we do coverage with PHP 7.4. There is some annoying issue with generating the coverage report with PHP 7.3

### CODE REVIEW

**Comment by JammingBen** [comment]
@phil-davis I tried increasing the `memory_limit` here for testing purposes. It's already quite high, but let's see if it changes anything. Next I should check what test(s) are responsible for that insance memory consumption.

**Comment by JammingBen** [comment]
I am once again clueless how to fix the issue with `bootstrap80.php` being loaded. Apparently it only affects PHP7.3 unit tests, PHP7.4 runs just fine.

I've started our docker images `owncloudci/php:7.3` and `owncloudci/php:7.4` locally and tried to run the tests, but I can't get them to succeed. The result is always `Generating code coverage report in Clover XML format ... Killed`. I was expecting at least the `owncloudci/php:7.4` container to run the tests successfully like Drone. @phil-davis Can you maybe try on your machine?

I have the feeling that it has something to do with the PHP7.3 Ubuntu runtime which is being used. Maybe `PHP_VERSION_ID` evaluates wrong with it.

Other than that, I have no idea what else to do. Of course we can skip coverage like we did in core for PHP7.3 in all affected repos.

**Comment by phil-davis** [comment]
I have the same problem in https://github.com/owncloud/files_classifier/pull/538

I suggest that we run the coverage pipelines with PHP 7.4 and just leave a single pipeline with PHP 7.3 and no ccoverage to demonstrate that the code works with PHP 7.3

File: .drone.star [hunk 1]
```diff
@@ -742,7 +742,7 @@ def phpTests(ctx, testType, withCoverage):
 
         for phpVersion in params["phpVersions"]:
             if testType == "phpunit":
-                if params["coverage"]:
+                if params["coverage"] and phpVersion not "7.3":
```
Comment (phil-davis) [inline .drone.star]: This is in common code that is exactly the same in all oC10 apps. I really don't want to change that - unless we decide to do implement this "rule" for every oC10 app. And that might be a bit confusing when someone explicitly sets "coverage" true and PHP 7.3 in the config section, but does not get any coverage.

File: .drone.star [hunk 2]
```diff
@@ -799,7 +799,7 @@ def phpTests(ctx, testType, withCoverage):
                     },
                 }
 
-                if params["coverage"]:
+                if params["coverage"] and phpVersion not "7.3":
```
Comment (phil-davis) [inline .drone.star]: Same comment here.

File: .drone.star (hunk 1 above)
Comment (JammingBen) [reply .drone.star]: Ahh okay. How do you suggest to skip the coverage then?

File: .drone.star [hunk 3]
```diff
@@ -742,7 +742,7 @@ def phpTests(ctx, testType, withCoverage):
 
         for phpVersion in params["phpVersions"]:
             if testType == "phpunit":
-                if params["coverage"]:
+                if params["coverage"] and phpVersion != "7.3":
```
Comment (phil-davis) [inline .drone.star:745]: This is in common code that is exactly the same in all oC10 apps. I really don't want to change that - unless we decide to do implement this "rule" for every oC10 app. And that might be a bit confusing when someone explicitly sets "coverage" true and PHP 7.3 in the config section, but does not get any coverage.

File: .drone.star [hunk 4]
```diff
@@ -799,7 +799,7 @@ def phpTests(ctx, testType, withCoverage):
                     },
                 }
 
-                if params["coverage"]:
+                if params["coverage"] and phpVersion != "7.3":
```
Comment (phil-davis) [inline .drone.star:802]: Same here

File: .drone.star (hunk 3 above)
Comment (JammingBen) [reply .drone.star:745]: Ahh okay. How do you suggest to skip the coverage then?

Comment (phil-davis) [reply .drone.star:745]: I will push something...

**Comment by phil-davis** [comment]
Oops - I reverted the wrong thing. I will try again...

**Review by phil-davis — APPROVED** [review]
I squashed the commits.

**Comment by JammingBen** [comment]
Looking good now. The memory limit is something we can check in the future, for now I'd say we're good to go.

**Pull request merged**